website/docs/docs/supported-databases/profile-redshift.md

```diff
@@ -71,6 +72,31 @@ my-redshift-db:
 
 </File>
 
+Ideally, the region and access keys should be configured in the usual `~/.aws` folder,
+via profiles you can then specify using the `iam_profile` option.
+
+A sample `~/.aws/config` file would be:
+<File name='~/.aws/config.yml'>
+
+```
+[profile data_engineer]
+role_arn=arn:aws:iam::<you-account-id>:role/Data_Engineer
+region = eu-west-1
+source_profile = default
+```
+
+</File>
+where the `source_profile` would be the profile that holds the access credentials, set in the `~/.aws/credentials` file
+
+<File name='~/.aws/config.yml'>
+
+```
+[default]  # 'default' profile
```

Comment:
Is the idea here that the `data_engineer` profile above is pointing back to the `default` profile for access keys? I think I don't totally understand the purpose of these docs, and I question if they're relevant enough to dbt to include here. If there's an external resource we can link out to that provides best practices around how to configure an `.aws/config` file, I'd be supportive of linking out to it here.